### DISCUSSION TITLE
blendModes with transparent background

### DISCUSSION THREAD

**Discussion by konserwa2** [pixijs/pixijs#10861] [General]
Hello,

Is it intended that when I apply blendMode other than normal, part of the sprite that should be transparent, becomes black?

You can see this effect when you set app background to transparent. Just visit pixi blendMode example (link below) and change `backgroundColor: 'white',` to `backgroundAlpha: 0,` in app.init() options.
https://pixijs.com/8.x/examples/basic/blend-modes

**Comment by photonstorm** [comment]
If there is no background color set, then yes. It can't blend with the page behind it, so it defaults to nothing.

**Reply by konserwa2** [reply]
I just don't get why wouldn't it default to transparency?

**Reply by photonstorm** [reply]
If the background is fully transparent (RGBA = 0, 0, 0, 0), the RGB components are treated as black (0, 0, 0). Multiplying any color by black (0) results in black.

**Reply by konserwa2** [reply]
Yeah, but maybe it could not ignore alpha value?

Anyway, thank you for your help. I'll just assume blend modes are useless in case transparency is present xd
